install.sh

```diff
@@ -1967,7 +1393,37 @@ TM_SSH_FILES="src/tm_mad/ssh/clone \
               src/tm_mad/ssh/cpds \
               src/tm_mad/ssh/resize \
               src/tm_mad/ssh/ssh_utils.sh \
-              src/tm_mad/ssh/recovery_snap_create_live"
+              src/tm_mad/ssh/recovery_snap_create_live \
+              src/tm_mad/ssh/prebackup_live \
+              src/tm_mad/ssh/prebackup \
+              src/tm_mad/ssh/postbackup_live \
+              src/tm_mad/ssh/postbackup \
+              src/tm_mad/ssh/restore"
+
+TM_LOCAL_FILES="src/tm_mad/local/clone \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

TM_LOCAL_FILES appears unused. Verify use (or export if used externally).